zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java

```diff
@@ -274,24 +243,25 @@ public void onClick(View v) {
 	@Override
 	protected void onDestroy() {
 		super.onDestroy();
+		Headset.stop();
 		Zello zello = Zello.getInstance();
 		zello.unsubscribeFromEvents(this);
 		zello.unconfigure();
 		_audio = null;
-
-		Headset.stop();
 	}
 
 	@Override
 	protected void onResume() {
 		super.onResume();
 		Zello.getInstance().leavePowerSavingMode();
+		//Headset.onForeground();
```

Comment:
Assuming this isn't meant to stay this way - should you remove it entirely?

Reply:
Yep, forgot to push the last commit.